pkgs/games/steam/chrootenv.nix

```diff
@@ -178,6 +178,12 @@ in buildFHSUserEnv rec {
   '';
 
   profile = ''
+    # Workaround for issue #44254 (Steam cannot connect to friends network)
+    # https://github.com/NixOS/nixpkgs/issues/44254
+    test -z ''${TZ+x} &&
+    new_TZ="$(readlink -f /etc/localtime | grep -o '(?<=/zoneinfo/).*$' -P)" &&
```

Comment:
The `-P` argument to `grep` could go just after `grep` like `grep -P -o '...'`, that would likely make reading the code easier (as one would know that they are reading a PCRE regex before actually reading it)

Reply:
Thanks for the advice. I'll do that.